change/@ni-nimble-components-af9b983d-43db-46f7-a204-c271899c2559.json

```diff
@@ -1,5 +1,5 @@
 {
-  "type": "minor",
+  "type": "major",
   "comment": "Introducing data hierarchy to table",
```

Comment:
It might be helpful to be more specific about what the breaking change was in each of these change files. Something like _renamed `groupsCollapseAll` on the table label provider to `collapseAll`_. That will make it easier for someone scanning the changelog to see if they need to worry about a given breaking change.